app/templates/sensors.html

```diff
@@ -42,8 +42,30 @@
 				</ul>
 			</div>
 		</div>
-		<div class="container">
-
+		<div class="container" style="margin-top: 20px;">
+			<table class="table table-striped">
+			  <thead>
+			    <tr>
+			      <th scope="col">Id</th>
+			      <th scope="col">Mac Address</th>
+			      <th scope="col">Assigned Field</th>
+			    </tr>
+			  </thead>
+			  <tbody>
+			    <tr ng-repeat="sensor in sensors">
+			      <th scope="row">{a sensor.id a}</th>
+			      <td>{a sensor.mac a}</td>
+			      <td>
+			      	<div class="form-group">
+					    <select class="form-control" id="fieldSelect">
+					      <option>1</option>
```

Comment:
This needs to be dynamic from mongo. Let's hold off on approving this PR until that has been integrated

Reply:
I will be working this in my fixes to the Settings Page PR so stay tuned